src/SolverAPI.jl

```diff
@@ -174,26 +174,38 @@ gracefully and included in `Response`.
     the `MOI.ModelLike` model and should return `Nothing`.
   - `request`: A `SolverAPI.Request` specifying the optimization problem.
   - `solver`: A `MOI.AbstractOptimizer` to use to solve the problem.
+
+## Keyword Args
+
+  - `bridge_type`: The type to use for the solver cache and bridge. Either `Int` or `Float64`. Defaults to `Float64`.
 """
-function solve(fn, json::Request, solver::MOI.AbstractOptimizer)
+function solve(
+    fn,
+    json::Request,
+    solver::MOI.AbstractOptimizer;
+    bridge_type::Type = Float64,
```

Comment:
it is a bit more general than just a bridge type, it is the numeric type of the MOI model, so `numeric_type` might be a better name